code/modules/admin/admin_verbs.dm

```diff
@@ -812,17 +812,17 @@
 
 	if(AH && AH.tier == TICKET_ADMIN && !check_rights(R_ADMINTICKET, FALSE))
 		return
-
+	var/client/usr_client = usr.client
```

Comment:
It should be using key, not the client reference, this PR is a regression

Reply:
Reverting all this